### PR TITLE
USWDS SITE - Address Form

### PR DESCRIPTION
**Pull request by chazchumley**
## Preview
[Updated Guidance](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/cc-address-form-guidance/components/form-templates/#address-form)

## Description

Update guidance on Address Form Template changes to include an Additional Information section with a link to USPS guidance when using Address Forms.

## Additional information

* Added "Additional Information" section
* Added link to "USPS" guidance on using Address Forms.

### CODE REVIEW

**Comment by jAigret-Bix** [comment]
@afeijoo  need guidance on the second link. Should this be deleted or changed to a PDF? It is currently a gDoc
Screen Shot 2020-11-24 at 12 24 41 PM

File: _components/form-templates/02-address-form.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ lead: Enter a standard U.S. mailing or shipping address
     <ul class="usa-content-list">
       <li>As you customize this form template, make sure it continues to follow the <a href="{{ site.baseurl }}/form-templates/">accessibility guidelines for form templates</a> and the <a href="{{ site.baseurl }}/form-controls/">accessibility guidelines for form controls</a>.</li>
     </ul>
+    <ul class="usa-content-list">
+      <li>As you customize this form template, make sure it continues to follow the <a href="{{ site.baseurl }}/form-templates/">accessibility guidelines for form templates</a> and the <a href="{{ site.baseurl }}/form-controls/">accessibility guidelines for form controls</a>.</li>
+    </ul>
+    <h4>Additional Information</h4>
+    <p>
+      <a href="https://pe.usps.com/text/pub28/28c1_001.htm">USPS Publication 28</a> defines the mailing standards in the United States (see <a href="https://drive.google.com/file/d/1T1dmOfuBoOypUw4FbOSYtOHaTGUGE9MC/view?usp=sharing">this summary of applicable standards</a> 🔒 for easier reference and search)
```
Comment (afeijoo) [inline _components/form-templates/02-address-form.md:41]: Remove "(see <a href="https://drive.google.com/file/d/1T1dmOfuBoOypUw4FbOSYtOHaTGUGE9MC/view?usp=sharing">this summary of applicable standards</a> 🔒 for easier reference and search)"

**Review by afeijoo — APPROVED** [review]
LGTM with one minor suggestion for the additional guidance

**Comment by thisisdano** [comment]
Looks good — holding until I merge the `site` work

**Comment by mejiaj** [comment]
circleci fails on

```bash
Issues with no direct upgrade or patch:
  ✗ Prototype Pollution [High Severity][https://snyk.io/vuln/SNYK-JS-Y18N-1021887] in y18n@3.2.1
    introduced by gulp@4.0.2 > gulp-cli@2.3.0 > yargs@7.1.1 > y18n@3.2.1
  This issue was fixed in versions: 4.0.1, 5.0.5
```

There doesn't seem to be a fix currently.

**Comment by mejiaj** [comment]
Blocked until https://github.com/uswds/uswds/pull/3987 is merged.

**Comment by mejiaj** [comment]
Blocked by uswds/uswds/issues/3989

**Pull request merged**